Index: Dart/variable.dart
===================================================================
diff --git a/Dart/variable.dart b/Dart/variable.dart
deleted file mode 100644
--- a/Dart/variable.dart	(revision b763173622c9fc240919e93cfb7d9bc6414bb81f)
+++ /dev/null	(revision b763173622c9fc240919e93cfb7d9bc6414bb81f)
@@ -1,190 +0,0 @@
-  ///0 - comments section
-   // two forwardslash is for commenting or explain something or skipping any line in code & this only works in one line
-   // (/* - */) this method is for skipping a bunch of lines. EXAMPLE -
-    /*
-   print('Hello World');
-  print("Hello Dart");
-  print('''Hello Flutter''');
-    */
-//Starting with the function is responsible for all kinds of execution.
-  main() {
-  ///1 - Print Hello World
-
-  print('Hello World'); //using one quotation mark
-  print("Hello Dart"); //using two quotation mark
-  print('''Hello Flutter'''); //using three quotation mark
-
-  ///2 - Line breaking, quotation mark & use of apostrophe
-
-  print('Rahim\'s \nPen'); //using one quotation mark, backslash apostrophe (\') for apostrophe & backslash n (\n) for line break
-  print("Karim's \nPencil"); //using two quotation mark which is acceptable to use BASIC APOSTROPHE & backslash n (\n) for line break
-  print('''Rumi's
-     "Mobile"'''); //using three quotation mark which is also acceptable to use BASIC APOSTROPHE & we can use the BASIC SPACE & RETURN key to perform space & line break
-
-  ///2 - Variable & Arithmetic Operators
-
-  var x = 20; //holding 20 in the variable name x
-    print(x); //printing the variable x
-
-  var y = 5; //holding 5 in the variable name y
-    print(y); //printing the variable y
-
-  y = 10; //update 10 in the variable name y
-    print(y); //printing the variable y after updating the value
-
-  var z = x+y; //holding the sum of var x & var y in the variable name z
-    print(z); //printing the variable z
-
-  var p = x*z; //holding the multiplication of var x & var z in the variable name p
-    print(p); //printing the variable p
-
-  var q = p/20; //holding the division of var p & 20 in the variable name q
-    print(q); //printing the variable q
-
-  var r = 20%3; //holding the modulus of 20 & 3 in the variable name r
-    print(r); //printing the variable r
-
-
-
-
-
-
-  print(x+y);
-  print(x-y);
-  print(x*y);
-  print(x/y);
-  print(x%y);
-
-
-
-
-    ///3 - String & Boolean
-
-    /*
-  var myName = "It's Muj";
-  var myBname = 'Taw';
-
-  print(myName);
-  print(myBname);
-
-  var isMeMale = true;
-  var isMeFemale = false;
-
-  print(isMeMale);
-  print(isMeFemale);
-   */
-
-    ///4 - Dart List & List Properties
-
-/*
-  var bdDiv =
-  [
-    'Dhaka',
-    'Khulna',
-    'Barisal',
-    'Chittagong',
-    'Mymensingh',
-    'Rajshahi',
-    'Sylhet',
-    'Rangpur'
-  ];
- // var res = bdDiv.length;
-  //var res = bdDiv.last;
-  //var res = bdDiv.;
-
-
-
-  //print(bdDiv);
- // print(bdDiv[1]);
-  //print('${bdDiv[1]} and ${bdDiv[2]}');
-  //print(res);
-*/
-    ///5 - Dart Map
-/*
-  var wCouCap =
-  {
-    'Bandladesh' : 'Dhaka',
-    'India' : 'Delhi',
-    'Japan' : 'Tokyo'
-  };
-
-  print(wCouCap);
-  print(wCouCap['Japan']);
-
- */
-
-    ///6 - Dart Unary Operators
-    /*
-  var x = 50;
-  print(x++);
-  print(x++);
-  print(x++);
-  print(x++);
-  print(x++);
-
-
-  var xy = 50;
-  print(++xy);
-  print(++xy);
-  print(++xy);
-  print(++xy);
-  print(++xy);
-
-
-  var y = 300;
-  print(y++);
-  print(y++);
-  print(++y);
-  print(--y);
-  print(y--);
-
-
-  ///7 - Dart Constant
-
-  final w = 50000;
-  const e = 1700;
-
-  print(w-e*e/w);
-
-   */
-
-    ///8 - Dart Fixed Length List & Growable List
-    /*
-  var city=[
-    'Dhaka',
-    'Khulna',
-    'Barisal',
-  ];
-
-  print(city);
-
-  city.add('nepal');
-
-  print(city);
-
-  const cityy=[
-    'Dhaka',
-    'Khulna',
-    'Barisal',
-  ];
-
-  print(cityy);
-
-*/
-    ///9 - List insert
-
-    var num=[1, 2, 3,];
-
-    num[0]=600;
-
-    num.insert(2,10);
-    num.insertAll(2,[50000,520]);
-    num.removeAt(2);
-
-
-
-    print(num);
-
-
-
-  }
Index: Practice-Dart_MOD1.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Practice-Dart_MOD1.iml b/Practice-Dart_MOD1.iml
new file mode 100644
--- /dev/null	(date 1681459834104)
+++ b/Practice-Dart_MOD1.iml	(date 1681459834104)
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" name="Dart SDK" level="project" />
+  </component>
+</module>
\ No newline at end of file
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1681460562325)
+++ b/.idea/vcs.xml	(date 1681460562325)
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/Dart_Practice" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/libraries/Dart_SDK.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/libraries/Dart_SDK.xml b/.idea/libraries/Dart_SDK.xml
new file mode 100644
--- /dev/null	(date 1681459834177)
+++ b/.idea/libraries/Dart_SDK.xml	(date 1681459834177)
@@ -0,0 +1,27 @@
+<component name="libraryTable">
+  <library name="Dart SDK">
+    <CLASSES>
+      <root url="file:///usr/local/opt/dart/libexec/lib/async" />
+      <root url="file:///usr/local/opt/dart/libexec/lib/cli" />
+      <root url="file:///usr/local/opt/dart/libexec/lib/collection" />
+      <root url="file:///usr/local/opt/dart/libexec/lib/convert" />
+      <root url="file:///usr/local/opt/dart/libexec/lib/core" />
+      <root url="file:///usr/local/opt/dart/libexec/lib/developer" />
+      <root url="file:///usr/local/opt/dart/libexec/lib/ffi" />
+      <root url="file:///usr/local/opt/dart/libexec/lib/html" />
+      <root url="file:///usr/local/opt/dart/libexec/lib/indexed_db" />
+      <root url="file:///usr/local/opt/dart/libexec/lib/io" />
+      <root url="file:///usr/local/opt/dart/libexec/lib/isolate" />
+      <root url="file:///usr/local/opt/dart/libexec/lib/js" />
+      <root url="file:///usr/local/opt/dart/libexec/lib/js_util" />
+      <root url="file:///usr/local/opt/dart/libexec/lib/math" />
+      <root url="file:///usr/local/opt/dart/libexec/lib/mirrors" />
+      <root url="file:///usr/local/opt/dart/libexec/lib/svg" />
+      <root url="file:///usr/local/opt/dart/libexec/lib/typed_data" />
+      <root url="file:///usr/local/opt/dart/libexec/lib/web_audio" />
+      <root url="file:///usr/local/opt/dart/libexec/lib/web_gl" />
+    </CLASSES>
+    <JAVADOC />
+    <SOURCES />
+  </library>
+</component>
\ No newline at end of file
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
new file mode 100644
--- /dev/null	(date 1681459834198)
+++ b/.idea/misc.xml	(date 1681459834198)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager">
+    <output url="file://$PROJECT_DIR$/out" />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1681459838234)
+++ b/.idea/.gitignore	(date 1681459838234)
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
